config.json

```diff
@@ -1443,15 +1438,15 @@
     }
   ],
   "tags": [
+    "execution_mode/compiled",
     "paradigm/declarative",
     "paradigm/functional",
-    "typing/static",
-    "typing/strong",
-    "execution_mode/compiled",
-    "platform/windows",
-    "platform/mac",
     "platform/linux",
+    "platform/mac",
+    "platform/windows",
     "runtime/standalone_executable",
+    "typing/static",
+    "typing/strong",
     "used_for/backends",
     "used_for/financial_systems"
   ]
```

Comment:
A lot of this file has changed, which seems unrelated to the exercise being added.

Reply:
Running `configlet create --practice-exercise <slug>` formats the track config as if the user had also run `configlet fmt`. So if this was the first Haskell exercise added since that functionality was added, the noise makes sense. The next Haskell exercise PR though shouldn't have this noise since the config would be already correctly formatted.

Reply:
@BNAndras I defer to you! :)